test/dlc_tests/ec2/pytorch/training/test_pytorch_training.py

```diff
@@ -157,9 +155,8 @@ def test_pytorch_healthcheck_dcgm(pytorch_training, ec2_connection, gpu_only, ec
     execute_ec2_training_test(ec2_connection, pytorch_training, PT_DCGM_TEST_CMD)
 
 
-@pytest.mark.skip_pt20_cuda121_tests
-@pytest.mark.skip_pt21_test
-@pytest.mark.skip_pt22_test
+@pytest.mark.skip_serialized_release_pt_test
+@pytest.mark.skip_nccl_healthcheck_test
```

Comment:
looks like candidate for removal - what do you think @roywei?

If we want to keep it around, let's avoid adding a custom fixture and just pytest.skip directly with the messaging we want, to avoid obfuscating the meaning.